source/_posts/en/tutorial/style.md

```diff
@@ -31,11 +31,11 @@ san.defineComponent({
 });
 ```
 
-上面的例子中，isHidden 数据为真时，ul 具有 list-hidden 的 class，为假时不具有。
+In the above example, when the isHidden is `true`, ul has `list-hidden` class, which does not exist when isHidden is `false`.
```

Comment:
when the isHidden is `true`, ul has `list-hidden` class, which does not exist when isHidden is `false`.
`list-hidden` class appears when `isHidden` is `true`, otherwise it is omitted